gpt4all-chat/metadata/models.json

```diff
@@ -156,7 +164,8 @@
     "parameters": "13 billion",
     "quant": "q4_2",
     "type": "LLaMA",
-    "description": "<strong>Trained with RHLF by Stability AI</strong><br><ul><li>Instruction based<li>Cannot be used commercially</ul>"
+    "description": "<strong>Trained with RHLF by Stability AI</strong><br><ul><li>Instruction based<li>Cannot be used commercially</ul>",
+    "systemPrompt": "## Assistant: I am StableVicuna, a large language model created by CarperAI. I am here to chat!\n\n"
```

Comment:
systemPrompt with no promptTemplate here? per [HF demo](https://huggingface.co/spaces/CarperAI/StableVicuna/blob/main/app.py) it should be `"###Human: %1\n###Assistant:"`

```python
prompt_template = Template("""\
### Human: $human
### Assistant: $bot\
""")
```

Reply:
That's the default template which will be used when another isn't explicitly requested.